sample/src/main/java/org/aerogear/graphqlandroid/data/UserData.kt

```diff
@@ -139,6 +176,13 @@ class UserData(val context: Context) {
         }
 
         val customCallback = object : ResponseCallback {
+            override fun onConflictDetected(
+                serverState: String,
+                clientState: String
+            ): Mutation<Operation.Data, Any, Operation.Variables> {
+               return mutation as com.apollographql.apollo.api.Mutation<Operation.Data, Any, Operation.Variables>
```

Comment:
Here as well, return null if you don't want to handle conflicts.
Passing in the mutation object makes no sense design wise.